src/ufixed.jl

```diff
@@ -10,18 +10,34 @@ end
 
   rawtype{T,f}(::Type{UFixed{T,f}}) = T
 nbitsfrac{T,f}(::Type{UFixed{T,f}}) = f
+typechar{X<:UFixed}(::Type{X}) = 'N'
+signbits{X<:UFixed}(::Type{X}) = 0
+
+for T in (UInt8, UInt16, UInt32, UInt64)
+    for f in 0:sizeof(T)*8
+        sym = Symbol(takebuf_string(showtype(_iotypealias, UFixed{T,f})))
+        @eval begin
+            typealias $sym UFixed{$T,$f}
+            export $sym
+        end
+    end
+end
 
+# ASCII typealiases
+typealias U8       UFixed{UInt8,8}
 typealias UFixed8  UFixed{UInt8,8}
 typealias UFixed10 UFixed{UInt16,10}
 typealias UFixed12 UFixed{UInt16,12}
 typealias UFixed14 UFixed{UInt16,14}
 typealias UFixed16 UFixed{UInt16,16}
+typealias U16      UFixed{UInt16,16}
```

Comment:
I largely agree with this. But should we apply the same logic to `U8`? In the Colors/Images world, we use `RGB{U8}` all the time. That would become `RGB{N0f8}`, which is not bad, but a little more to remember. (Probably I'm just not used to it, though.)

I also think it's valid to decide to keep U8 but ditch U16. Basically, I am persuadable to any outcome on this point :smile:.